coordinator/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -102,7 +102,7 @@ private synchronized void recoverTransactions() {
     }
 
     private void doRecoverTransaction(Uid recoverUid) {
-        // Retrieve the transaction status from its original process.
+        // Retrieve the transaction status from its original process // TODO remove because it is not needed
```

Comment:
Do you also want to track this TODO with a JBTM?

Reply:
Good plan